homeassistant/components/openuv/__init__.py

```diff
@@ -187,26 +354,33 @@ class OpenUvEntity(Entity):
 
     def __init__(self, openuv: OpenUV, description: EntityDescription) -> None:
         """Initialize."""
+        coordinates = f"{openuv.client.latitude}, {openuv.client.longitude}"
         self._attr_extra_state_attributes = {}
         self._attr_should_poll = False
-        self._attr_unique_id = (
-            f"{openuv.client.latitude}_{openuv.client.longitude}_{description.key}"
-        )
+        self._attr_unique_id = f"{coordinates}_{description.key}"
```

Comment:
Looks like the unique_id is changed from using underscore to separate the coordinates to comma space.

Reply:
Good catch! I'll get that fixed in a new PR.

Reply:
https://github.com/home-assistant/core/pull/78691